web/src/javascript/drawMap.js

```diff
@@ -138,9 +140,31 @@ function getMousePos(evt) {
   };
 }
 
-screenCanvas.addEventListener("click", function (evt) {
-  const mousePos = getMousePos(evt);
+function mouseClick(e) {
+  const mousePos = getMousePos(e);
+  if (mousePos.x > miniMapPosX && mousePos.y < miniMapHeight) {
+    changeScreenPos(mousePos);
+  }
+}
+
+screenCanvas.onmousedown = function(e) {
+  mouseIsDown = true;
+  const mousePos = getMousePos(e);
+  if (mousePos.x > miniMapPosX && mousePos.y < miniMapHeight) {
```

Comment:
Ideally it would be nice if this was part of a common function, like `minimapClick` or similar. It would allow us to easily change the position of the mini-map, for example.